src/corelibs/U2Core/src/models/GObject.cpp

```diff
@@ -321,7 +322,7 @@ void GObject::removeRelations(const QString &removedDocUrl) {
             changed = true;
         }
     }
-    if (changed) {
+    if (changed && QFileInfo::exists(removedDocUrl)) {
```

Comment:
You know this is a hack and you know I do not approve hacks.
Removal of relations from object relations map does not depend on existence or non-existence of the related file at all.

Reply:
Yes, my bad, forget it. 

The problem here is that `setObjectRelations(rels);` needs to create a connection to the existed database, which is the current `GObject` has reference to (the path to the database file is stored in `GObject::entityRef::dbiRef::dbiId`) . As far as I can understand, it anticipates, that this database would be the session one, wich is stashed far-far on the disk and shouldn't be deteled by user. But, we have a separate database file here, which could be deleted by user and `GObject` doesn't expect it. I can check that the database file exists, but not sure that it's the solution you want to see here. 

So, what the best solution you need?

Reply:
I replaced "alignment.ugenedb" to "murine.ugenedb", cuz this problem doesn't appear on the first file (because "alignment.ugenedb" doesn't have any relations, but "murine.ugenedb" has related sequence and annotated table objects)

Reply:
I do not know a good solution here right now. Let we 1 day to dig into the problem and tomorrow I will suggest a solution

Reply:
OK. You could just build this branch, run the test and the problem SAFE_POINT will apear

Reply:
Here is my 'minimal-fix' suggestion:
```
@@ -159,7 +159,7 @@ void GObject::setObjectRelations(const QList<GObjectRelation> &list) {
 void GObject::setRelationsInDb(QList<GObjectRelation> &list) const {
     U2OpStatus2Log os;
     DbiConnection con(entityRef.dbiRef, os);
-    SAFE_POINT_OP(os, );
+    CHECK_OP(os, );  // Database is not available for some reason. It is may be deleted.
```

A complete fix (out of scope in this PR) must check all DB operations for successful completion like IO ops.

Reply:
OK